EWSType/FindMailboxStatisticsByKeywordsResponseMessageType.php

```diff
@@ -32,43 +19,4 @@ class EWSType_FindMailboxStatisticsByKeywordsResponseMessageType extends EWSType
      * @var EWSType_MailboxStatisticsSearchResultType
      */
     public $MailboxStatisticsSearchResult;
-
-    /**
-     * Provides a text description of the status of the response.
-     *
-     * @since Exchange 2013
```

Comment:
Oh, nooooo, I was just saying to remove the `@since` tags from the `EWSType_ResponseMessageType` since they'd no longer be accurate and wouldn't make sense as a base-class. The others are fine and quite helpful.